homeassistant/components/telegram_bot/__init__.py

```diff
@@ -597,7 +606,7 @@ def delete_message(self, chat_id=None, **kwargs):
         message_id, _ = self._get_msg_ids(kwargs, chat_id)
         _LOGGER.debug("Delete message %s in chat ID %s", message_id, chat_id)
         deleted = self._send_msg(
-            self.bot.deleteMessage, "Error deleting message", chat_id, message_id
+            self.bot.delete_message, "Error deleting message", None, chat_id, message_id
```

Comment:
The `camelCase` methods are just aliases for the `snake_case` methods, so I decided to use the original methods for consistency.